README.md

```diff
@@ -43,7 +44,7 @@ tools.
 - The **build.sh** is a script that calls upon gradle build to build the entire project which creates an executable to be able to launch the
 project in production mode. To run **build.sh**, go to your project directory in a terminal and enter:``./build ``
 
-When **build.sh** is run the script **.sage_run.sh** is copied to ~/**sage.sh** that when launched i.e. ``./sage.sh``
+When **build.sh** is run the script **.3601_run.sh** is copied to ~/**3601.sh** that when launched i.e. ``./3601.sh``
```

Comment:
…copied to `~/**3601.sh**`. When this is launched with, for example, `./3601.sh`, it will run your project…

Does that help clarify things some?